diff --git a/arch/arm64/boot/dts/qcom/sm8250-xiaomi-alioth.dts b/arch/arm64/boot/dts/qcom/sm8250-xiaomi-alioth.dts
index 2eff2c6b4..9ed7b4025 100644
--- a/arch/arm64/boot/dts/qcom/sm8250-xiaomi-alioth.dts
+++ b/arch/arm64/boot/dts/qcom/sm8250-xiaomi-alioth.dts
@@ -50,8 +50,8 @@ chosen {
 
 	battery: battery {
 		compatible = "simple-battery";
-		charge-full-design-microamp-hours = <4520000>;
-		energy-full-design-microwatt-hours = <17500000>;
+		charge-full-design-microamp-hours = <5000000>;
+		energy-full-design-microwatt-hours = <19000000>;
 		voltage-min-design-microvolt = <3400000>;
 		voltage-max-design-microvolt = <4460000>;
 	};
