--- a/arch/arm64/boot/dts/qcom/sm8250-xiaomi-alioth.dts
+++ b/arch/arm64/boot/dts/qcom/sm8250-xiaomi-alioth.dts
@@ -1,6 +1,6 @@
 	battery: battery {
 		compatible = "simple-battery";
-		charge-full-design-microamp-hours = <4520000>;
-		energy-full-design-microwatt-hours = <17500000>;
+		charge-full-design-microamp-hours = <5000000>;
+		energy-full-design-microwatt-hours = <19000000>;
 		voltage-min-design-microvolt = <3400000>;
 		voltage-max-design-microvolt = <4460000>;
 	};
